src/transformers/models/albert/modeling_albert.py

```diff
@@ -728,7 +728,7 @@ def forward(
 
         extended_attention_mask = attention_mask.unsqueeze(1).unsqueeze(2)
         extended_attention_mask = extended_attention_mask.to(dtype=self.dtype)  # fp16 compatibility
-        extended_attention_mask = (1.0 - extended_attention_mask) * -10000.0
+        extended_attention_mask = (1.0 - extended_attention_mask) * torch.finfo(self.dtype).min
```

Comment:
imho, the incantation is obtuse enough to make a utility function:

```python
def near_neg_inf(dtype):
    return torch.finfo(self.dtype).min
```

Reply:
I personally prefer to avoid those kinds of one-line functions as it makes it harder to read the code, with the reader having to navigate.